MediaRecorder.html

```diff
@@ -442,6 +443,46 @@
               <em>Return type:</em> <code>boolean</code>
             </div>
           </dd>
+
+          <dt><dfn><code>getSupportedTypes</code></dfn>, static</dt>
+          <dd>
+            When this method is invoked, the User Agent must run the following
+            steps:
+            <ol>
+            <li>Let <var>p</var> be a promise.</li>
+            <li>Run the following steps in parallel:
+             <ol>
+             <li>Let <var>resultList</var> be an empty list.
+             <li>For each <a href="https://tools.ietf.org/html/rfc2046">MIME
+             type</a> <var>type</var> that is supported for encoding and/or
+             multiplexing (i.e. <code>isSuportedType(<var>type</var>)</code>
+             is true), add <var>type</var> to <var>resultList</var>.
+
+             <var>type</var> MUST be a <a def-id="mime-types">valid MIME type
+             </a> [[!HTML5]]. The codecs parameter [[!RFC6381]], if necessary to
+             specify exactly how the resource is encoded, SHOULD be added
+             to <code>type</code>.
+             If <var>type</var> can be expressed in several equivalent ways
+             (e.g. with or without quotes and whitespace, or different
+             capitalization), the shortest form with the largest number of
+             lowercase characters MUST be used.
+
+             </a>.
+             <var>resultList</var> MAY be ordered by the UA so that MIME types
```

Comment:
MAY is too wishy washy.  If the intent is to declare preferences, then say that: i.e.

> the <a>user agent</a> MUST place codecs that it prefers before other codes